src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1100px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
+
+.header__navigation {
+  padding: 11px 0;
+}
+
+.navigation__list {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 0;
+}
+
+.item {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+  padding: 23px 0;
+  cursor: pointer;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
when you use color more than 1 time better to create variable and use variable instead